### DISCUSSION TITLE
Doorbell

### DISCUSSION THREAD

**Discussion by Jozef73** [andrei-tatar/node-red-contrib-smartnora#213] [Q&A]
Dear Andrei,

I can't trigger Doorbell in my phone to show notifications in Google Home App.
Sending payload from NodeRed (as your example) don't work, successfuly injected.
Voice notification is turned ON and Doorbell assigned in my house/room.
Doorbell presented in GH.

Any advice?

thank you

jozef

**Comment by andrei-tatar** [comment]
Issue already created. See answer over there https://github.com/andrei-tatar/node-red-contrib-smartnora/issues/214
